### PR TITLE
Improve site documentation

### PR DESCRIPTION
**Pull request by charleswhchan**
Edits:
- Add constants.py link.
- Clarify "command_warnings".
- Update "Intro to Playbook" to improve readability. The following sections are updated:
  - YAML dictionary
  - module idempotent
  - name attribute
  - handlers

### CODE REVIEW

**Comment by charleswhchan** [comment]
ping: Like? Dislike? Any comments?

File: docsite/rst/playbooks_intro.rst [hunk 1]
```diff
@@ -248,10 +247,9 @@ The goal of each task is to execute a module, with very specific arguments.
 Variables, as mentioned above, can be used in arguments to modules.
 
 Modules are 'idempotent', meaning if you run them
-again, they will make only the changes they must in order to bring the
-system to the desired state.  This makes it very safe to rerun
-the same playbook multiple times.  They won't change things
-unless they have to change things.
```
Comment (bcoca) [inline docsite/rst/playbooks_intro.rst:254]: though redundant, the last phrase is worth reiterating

**Comment by bcoca** [comment]
just one comment, still this needs a rebase, also if you could flatten the commits it would be helfpul.

**Comment by charleswhchan** [comment]
Thanks @bcoca. 
See PR: https://github.com/ansible/ansible/pull/15172

**Pull request closed without merging**